src/js/builtins/Module.ts

```diff
@@ -76,9 +76,7 @@ export function overridableRequire(this: CommonJSModuleRecord, id: string) {
     // If we can pull out a ModuleNamespaceObject, let's do it.
     if (esm?.evaluated && (esm.state ?? 0) >= $ModuleReady) {
       const namespace = Loader.getModuleNamespaceObject(esm!.module);
-      return (mod.exports =
```

Comment:
cc @paperdave @dylan-conway please take a look at this

Reply:
what is the reasoning behind this change. it is breaking.

Reply:
Discussed IRL. The goal of this, originally, was to let you mutate values on the ES module namespace object. But, that doesn't work and has not worked.

Instead, this has the effect of making console.log and (...) not work as well